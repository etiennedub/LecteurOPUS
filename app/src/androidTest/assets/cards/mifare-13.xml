<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">047C5BAB</List><List index="1">2A0C4A80</List><List index="2">EC48E000</List><List index="3">80000000</List><List index="4">00000000</List><List index="5">32164220</List><List index="6">6FA5DF50</List><List index="7">BE348D9F</List><List index="8">01BE9948</List><List index="9">A5802100</List><List index="10">650037D2</List><List index="11">0010D58E</List><List index="12">01BE9948</List><List index="13">94801100</List><List index="14">650037D2</List><List index="15">0010FB79</List></Pages></Mifare>