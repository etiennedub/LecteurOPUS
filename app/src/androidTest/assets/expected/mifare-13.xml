<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="25/07/2016" Time="22:04" Transfert="false" OperatorId="8" BusId="2" OperatorName="STM" BusName="" /><List index="1" Date="25/07/2016" Time="19:48" Transfert="false" OperatorId="8" BusId="1" OperatorName="STM" BusName="" /></Trip><Contract><List index="0" Subscription="true" Valid="false" NbTicket="0" OperatorId="8" OperatorName="STM" /></Contract></Card>
